src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -31,6 +32,12 @@
     public static final String VALID_NAME_BOB = "Bob Choo";
     public static final String VALID_EMAIL_AMY = "amy@example.com";
     public static final String VALID_EMAIL_BOB = "bob@example.com";
+    public static final String VALID_ADDRESS_AMY = "Block 312, Amy Street 1";
+    public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
+    public static final String VALID_REMARK_AMY = "Like skiing.";
+    public static final String VALID_REMARK_BOB = "Favourite pastime: Eating";
+    public static final String VALID_TAG_HUSBAND = "husband";
+    public static final String VALID_TAG_FRIEND = "friend";
```

Comment:
I find and replace all occurrence of `VALID_TAG_HUSBAND` to `VALID_TAG_INTERNATIONAL` and `VALID_TAG_FRIEND` to `VALID_TAG_LOCAL`, you might want to consider that?
```suggestion
    public static final String VALID_TAG_INTERNATIONAL = "international";
    public static final String VALID_TAG_LOCAL = "local";
```